test/parallel/test-console-is-a-namespace.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+// https://heycam.github.io/webidl/#es-namespaces
+// https://console.spec.whatwg.org/#console-namespace
+// https://github.com/w3c/web-platform-tests/blob/40e451c/console/console-is-a-namespace.any.js
+
+const common = require('../common');
+const assert = require('assert');
+const { test, assert_equals, assert_true, assert_false } = common.WPT;
+
+assert.doesNotThrow(() => {
+  global.console = global.console;
+});
+
+// Tests above are not from WPT.
+
+/* eslint-disable */
```

Comment:
If Node collaborators should not modify these tests in any way, please add a comment here stating that.